Makefile

```diff
@@ -187,7 +187,7 @@ build-postgres-operator-image: build/postgres-operator/Dockerfile
 ##@ Test
 .PHONY: check
 check: ## Run basic go tests with coverage output
-	$(GO_TEST) -cover ./...
+	QUERIES_CONFIG_DIR="$(CURDIR)/${QUERIES_CONFIG_DIR}" $(GO_TEST) -cover ./...
```

Comment:
:thinking: is there a way you can write the go tests so that you don't need access to the queries?

Reply:
Hmm... So I added this because one of the tests was failing without it, but I'm realizing that this test was being run as a unit test, but it's really an integration test. So, I'm thinking I should just add the following to the top of the `exporter_test.go` file:
```
//go:build envtest
// +build envtest
```

Reply:
:memo: we don't have to provide build tags in both of these syntaxes. Since [go1.18](https://tip.golang.org/doc/go1.18#go-build-lines),  `go:build` is the new, preferred syntax and `+build` can be able to be removed.

I see that we have them in a bunch of places still - maybe it's worth a story to remove them